rosie/rosie/chamber_of_deputies/tests/test_adapter.py

```diff
@@ -82,3 +82,11 @@ def test_update(self, reimbursements, fetch):
             call()()
         ))
         fetch.assert_called_once_with(Adapter.COMPANIES_DATASET, self.temp_path)
+
+    @freeze_time('2010-11-12')
+    @patch('rosie.chamber_of_deputies.adapter.fetch')
+    @patch('rosie.chamber_of_deputies.adapter.Reimbursements')
+    def test_coerce_situation_date(self, reimbursements, fetch):
+        adapter = Adapter(self.temp_path)
+        df = adapter.dataset
+        self.assertTrue(any(dt.month == 9 and dt.day == 6 for dt in df['situation_date']))
```

Comment:
In find this assertion kind of confusing… does it says that at leas one of the dates in `df['situation_date']` should be YYYY-09-06? Why are just testing the day and month and not the year? I think something like that would be more expressive for example:

```python
self.assertIn(date(2012, 9, 6), df.situation_date)
```

Reply:
Thanks for a outsider view of my code :)